compiler/test-resources/repl/reset-command

```diff
@@ -0,0 +1,15 @@
+scala> def f(thread: Thread) = thread.stop()
+there were 1 deprecation warning(s); re-run with -deprecation for details
+def f(thread: Thread): Unit
+
+scala>:reset -deprecation
```

Comment:
maybe one more test to make sure that double quotes are supported as expected? since you used `config.CommandLineParser` and not just `String.split`

and a test that plain `:reset` with no args still works?